### PR TITLE
(DOCSP-28553): Release notes - 16 March 2023

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Release notes - 16 March 2023

### Jira

- https://jira.mongodb.org/browse/DOCSP-28553

### Staged Changes

- [Backend Release Notes](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-28553/release-notes/backend/)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-realm update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/release-notes/backend.txt [hunk 1]
```diff
@@ -13,6 +13,22 @@ Atlas App Services Changelog
 2023 Releases
 -------------
 
+.. _backend_20230315:
+
+15 March 2023 Release
+~~~~~~~~~~~~~~~~~~~~~
+
+- Expiration times for user refresh tokens can be :ref:`configured from the UI
+  <TODO: DOCSP-27081>`.
```
Comment (mongodben) [inline source/release-notes/backend.txt:22]: will add ref once know what it is

File: source/release-notes/backend.txt [hunk 2]
```diff
@@ -13,6 +13,22 @@ Atlas App Services Changelog
...
+- Change the email address associated with email/password authentication via an
+  :admin-api-endpoint:`Admin API endpoint <operation/adminResetUserEmail>`
+- :ref:`Device Sync is automatically paused <mongodb-service-limitations-sync-inactive-pause>`
+  if it is inactive for 30 days.
+- Updated all existing apps using Atlas Device Sync with Flexible Sync to
```
Comment (dacharyc) [inline source/release-notes/backend.txt:27]: I have one minor question about wording for this one. Is this effectively an automatic migration to a new app/config structure? I wonder if we want to link to [the "Unified Rules System" section](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-28553/rules/sync-compatibility/#unified-rules-system) on that page, which goes into detail about the changes, and be more explicit that "Updated all existing apps" was some sort of automatic migration.

**Review by dacharyc — APPROVED** [review]
Overall LGTM. Just one small Q about wording, and obviously pending the TODO ref for configuring expiration times for user refresh tokens.

File: source/release-notes/backend.txt (hunk 1 above)
Comment (mongodben) [reply source/release-notes/backend.txt:22]:
```suggestion
  <refresh-token-expiration>`.
```

File: source/release-notes/backend.txt (hunk 2 above)
Comment (mongodben) [reply source/release-notes/backend.txt:27]: sounds good to me. didn't know that page existed 😄

File: source/release-notes/backend.txt [hunk 3]
```diff
@@ -13,6 +13,22 @@ Atlas App Services Changelog
...
+  :ref:`use the same rules and permissions <device-sync-compatibility>` as other services.
```
Comment (mongodben) [inline source/release-notes/backend.txt:28]:
```suggestion
  :ref:`use the same rules and permissions as other services <unified-rule-migration>`.
```

**Pull request merged**